proposals/raster_to_vector.json

```diff
@@ -0,0 +1,53 @@
+{
+    "id": "raster_to_vector",
+    "summary": "Converts a raster to a vector",
+    "description": "Converts a raster data cube into a vector data cube. The bounding polygon of similar areas of pixels is constructed.",
+    "categories": [
+        "cubes"
+    ],
+    "experimental": true,
+    "parameters": [
+        {
+            "name": "data",
+            "description": "A data cube.",
+            "schema": {
+                "type": "object",
+                "subtype": "datacube"
+            }
+        },
+        {
+            "name": "projection",
+            "description": "Coordinate reference system to reproject the converted vector. Specified as an [EPSG code](http://www.epsg-registry.org/) or [WKT2 CRS string](http://docs.opengeospatial.org/is/18-010r7/18-010r7.html).\nThe default value is of the  input raster cube.",
```

Comment:
> The default value is of the input raster cube.

doesn't sound very clean. I think it should be something like

> When no projection (or null) is given, the reference system of the input raster data cube is used

Reply:
Adopting the suggestion:)